FinSearchUnified/Tests/Helper/StaticHelperTest.php

```diff
@@ -119,14 +237,14 @@ public static function configDataProvider()
                 'isCategoryPage' => true,
                 'expected' => true
             ],
-            'FINDOLOGIC is active in search' => [
+            'FINDOLOGIC is active on search page' => [
                 'ActivateFindologic' => true,
                 'ShopKey' => '0000000000000000ZZZZZZZZZZZZZZZZ',
                 'ActivateFindologicForCategoryPages' => false,
                 'findologicDI' => false,
                 'isSearchPage' => true,
                 'isCategoryPage' => false,
-                'expected' => false
+                'expected' => true
```

Comment:
This case wasn't fixed, but rather the condition was changed. Make sure this still returns `false`. You may need some more setup so the `Environment` returns what you need.